src/modules/Dropdown/Dropdown.d.ts

```diff
@@ -216,6 +216,9 @@ export interface DropdownProps {
   /** A shorthand for a search input. */
   searchInput?: any;
 
+  /** Whether the searchQuery is cleared or not when selecting an item from search. */
+  searchQueryClearOnSelect?: boolean;
```

Comment:
not a nit at all, was hoping for some specific feedback on the naming anyway. i just put it near the search ones as it felt grouped, but really was unsure about naming convention. will switch to your suggestion.